pkg/api/validation/validation.go

```diff
@@ -727,7 +736,7 @@ func ValidateMinionUpdate(oldMinion *api.Node, minion *api.Node) errs.Validation
 
 	if !api.Semantic.DeepEqual(oldMinion, minion) {
 		glog.V(4).Infof("Update failed validation %#v vs %#v", oldMinion, minion)
-		allErrs = append(allErrs, fmt.Errorf("update contains more than labels or capacity changes"))
+		allErrs = append(allErrs, fmt.Errorf("update from %#v to %#v contains more than labels or capacity changes", oldMinion, minion))
```

Comment:
I'm not sure on this one - that will dump internal structures, which the user can't really do anything with.  Might be nice to generate a list of fields that are different, but I'd say it is not important yet.  Leave a TODO